include/umf/base.h

```diff
@@ -63,29 +63,23 @@ typedef enum umf_ctl_query_type {
 ///
 /// @brief Get value of a specified attribute at the given name.
 /// @param name name of an attribute to be retrieved
-/// @param ctx pointer to the pool or the provider
```

Comment:
Is the information on used params documented somewhere after these params description removal? I'm not sure if this is a good idea to place variadic arguments in this API. This approach would certainly require a very good documentation clearly stating how arguments are currently parsed.

What could be future arguments? Perhaps you could simply add unused `void *extra_args` for handling future ctl usage?

Reply:
I agree on adding documentation later, after more of CTL implementation is done.

`extra_args` would be a replacement for variadic args that would allow to have a clear Doxygen comment on possible args. But if you would document them later anyway, it's better to stick with variadic args.